src/dreyfus_index_updater.erl

```diff
@@ -64,7 +73,7 @@ update(IndexPid, Index) ->
         couch_db:close(Db)
     end.
 
-load_docs(FDI, _, {I, IndexPid, Db, Proc, Total, LastCommitTime}=Acc) ->
+load_docs(FDI, {I, IndexPid, Db, Proc, Total, LastCommitTime}=Acc) ->
```

Comment:
This change is used for pluggable storage engine API. The purge issue is based on plugable storage engine API.
But I think it should be commit separately.